frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx

```diff
@@ -109,6 +110,11 @@ export const TableNode: FC<Props> = ({ data: { table } }) => {
                         styles.handleCardinality,
                         styles.handleCardinalityRight,
                       ])}
+                      color={
+                        isHighlighted
+                          ? 'var(--node-layout)'
+                          : 'var(--global-border)'
+                      }
```

Comment:
This should also be set in the CSS Module!

Reply:
Thank you for comment!
I'll be careful next time👍
cd70245fba9cf6f533a08aebd8ae301ed3122819